test/runtests.jl

```diff
@@ -1,6 +1,9 @@
 using HDF5
 using Compat.Test
 using Compat.Distributed
+@static if VERSION ≥ v"0.7.0-DEV.3637"
```

Comment:
Is the `@static` is necessary here since it's in global scope?

Reply:
As far as I understand, `@static` means "evaluate the conditional at parse time". It shouldn't be related to the scope.